src/docs/development/ui/navigation/deep-linking.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Deep linking
+description: Navigate to routes when the app receives a new URL
+---
+
+{{site.alert.note}}
+  This feature is only available on the dev or master channel. See [Switching
+  Flutter channels][switching-channels] for more information.
```

Comment:
It's always best to say WHY before you say SEE. (It's even in our developer doc style guide).

```suggestion
  This feature is only available on the dev or master channel. To learn how to switch
  channels, see [Switching Flutter channels][switching-channels].
```